docs/.vuepress/components/metadata.vue

```diff
@@ -1,43 +1,117 @@
 <template>
   <header class="metadata" v-if="!isHomePage">
-    <span class="last-updated">
-      Last updated: <time :datatime="prettyDate">{{prettyDate}}</time>
+    <span v-if="updatedSinceLastVisit" class="last-updated last-updated--message">
+      Heads up! This page has been updated since your last visit {{ daysSinceLastVisit }} days ago
```

Comment:
`daysSinceLastVisit` seems like a computed property?

Reply:
It is now 👍